src/Microsoft.VisualStudio.Editors/SettingsDesigner/DataGridViewUITypeEditorColumnAccessible.vb

```diff
@@ -0,0 +1,5 @@
+﻿' Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+Public Class DataGridViewUITypeEditorColumnAccessible
```

Comment:
Was this file added intentionally? It looks like it could be deleted (which would fix the build failure):

```
RS0016: Symbol 'DataGridViewUITypeEditorColumnAccessible' is not part of the declared API
RS0016: Symbol 'implicit constructor for 'DataGridViewUITypeEditorColumnAccessible'' is not part of the declared API
```

Reply:
Ah this shouldn't be here, it was unintentional, thanks for catching it.